.travis.yml

```diff
@@ -16,10 +17,17 @@ jobs:
     - stage: test
       name: "Build and deploy"
       script: bash travis_test
+      install:
+        - wget https://releases.hashicorp.com/terraform/"$TF_VERSION"/terraform_"$TF_VERSION"_linux_amd64.zip
+        - unzip terraform_"$TF_VERSION"_linux_amd64.zip
+        - sudo mv terraform /usr/local/bin/
+        - rm terraform_"$TF_VERSION"_linux_amd64.zip
+        - terraform init
       before_deploy:
         - bash docker_push
+        - terraform apply -auto-approve -var "access_key=$ACCOUNT_KEY" -var "github_oauth_client_id=$OAUTH_CLIENT_ID" -var "github_oauth_client_secret=$OAUTH_CLIENT_SECRET"
       deploy:
```

Comment:
You might find that this stage errors when running because Travis cleans up the terraform config that terraform init has generated by default before this stage - if that is an issue you might want to use the [skip_cleanup](https://docs.travis-ci.com/user/deployment/#uploading-files-and-skip_cleanup) flag

Reply:
done